### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by YuvalBahat** [ImagingLab/ICIAR2018#4]
Hi,
I am wondering about the validation set you are using in your code. From what I could see, neither the [older dataset](https://rdm.inesctec.pt/dataset/nis-2017-003) nor the [ICIAR 2018 dataset](https://iciar2018-challenge.grand-challenge.org/Dataset/) include a validation set. Can you elaborate on how you obtained/created this sub-set?
Thanks a lot!

**Comment by knazeri** [comment]
@YuvalBahat We randomly chose 20% of the training set for the validation set and later used 10-fold cross validation across 10 model ensembles!

**Comment by YuvalBahat** [comment]
@knazeri Thanks for your prompt reply!
